pkg/events/events.go

```diff
@@ -6263,6 +6264,21 @@ var Definitions = eventDefinitions{
 				{Type: "dev_t", Name: "dev"},
 			},
 		},
+		FailedExecution: {
+			ID32Bit: sys32undefined,
+			Name:    "failed_execution",
```

Comment:
All events should have the `DocPath` field pointing to the relevant documentation path.
Please create a documentation file for this event and put its path here.